Gemfile

```diff
@@ -3,9 +3,7 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in manageiq-messaging.gemspec
 gemspec
 
-case ENV['TEST_RAILS_VERSION']
-when "6.0"
-  gem "activesupport", "~>6.0.4"
-when "6.1"
-  gem "activesupport", "~>6.1.4"
-end
+version = ENV.fetch('TEST_RAILS_VERSION', '6.1')
+raise "Unsupported version of Rails: #{version}" unless %w[6.0 6.1 7.0].include?(version)
+
+gem "activesupport", "~>#{version}", "< #{version.to_f + 0.1}"
```

Comment:
This floating point math gets weird if we wanted to test with a more specific rails version:
```
>> '7.1.3.0'.to_f + 0.1
=> 7.199999999999999
```

Reply:
yeah, that's why it raises.  It's only ever been for major/minor versions.  The prior mechanism led to hardcoded stale versions, such as 6.0.4

Reply:
>  The prior mechanism led to hardcoded stale versions, such as 6.0.4

Sort of? It would install the latest patch in that z stream, so it wasn't old so much as a minimum.

Reply:
Right, it would never test with 6.0.5-6.0.6.1, or 6.1.5-6.1.7.6.  I assumed we wanted to always pull in new patch versions but not minor versions.

I could pull in rubygems versioning methods but wanted to keep it super simple.

Let me know what the intent was here and I'll update it.

Reply:
Nope, I misread it.

Reply:
I'll resurrect it and just accept that code climate doesn't like multiple gem lines for the same gem in the gemspec.

Reply:
Ok, thanks for pointing out what I misread.  I pushed an update that should appease code climate and hardcode the 6.1 default instead of relying on the upper bound version constraint picking 6.1.